Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -1957,6 +1962,37 @@ + (void)track:(NSString *)event withValue:(double)value andInfo:(NSString *)info
     [[LPCountAggregator sharedAggregator] incrementCount:@"track"];
 }
 
++ (void)trackGeofence:(NSString *)event withInfo:(NSString *)info {
+    if ([[LPFeatureFlagManager sharedManager] isFeatureFlagEnabled:@"track_geofence"]) {
+        [self trackGeofence:event withValue:0.0 andInfo:info andArgs:nil andParameters:nil];
+    } else {
+        [[LPCountAggregator sharedAggregator] incrementCount:@"track_geofence_disabled"];
+    }
+}
+
++ (void)trackGeofence:(NSString *)event withValue:(double)value andInfo:(NSString *)info
+                           andArgs:(NSDictionary *)args andParameters:(NSDictionary *)params
+{
+    RETURN_IF_NOOP;
+    LP_TRY
```

Comment:
can you help me understand the reason for these 2 constants or macros?

Reply:
I copied them from the `track` method. I think they were probably added after experiencing issues/crashes, not sure of the exact reason.